src/helics/application_api/FederateInfo.cpp

```diff
@@ -195,6 +197,12 @@ static constexpr frozen::unordered_map<std::string_view, int, 89> flagStringsTra
     {"single_thread_federate", HELICS_FLAG_SINGLE_THREAD_FEDERATE},
     {"singlethreadfederate", HELICS_FLAG_SINGLE_THREAD_FEDERATE},
     {"singleThreadFederate", HELICS_FLAG_SINGLE_THREAD_FEDERATE},
+    {"single_thread_core", HELICS_FLAG_SINGLE_THREAD_CORE},
+    {"singlethreadcore", HELICS_FLAG_SINGLE_THREAD_CORE},
+    {"singleThreadcore", HELICS_FLAG_SINGLE_THREAD_CORE},
+    {"multi_thread_core", HELICS_FLAG_MULTI_THREAD_CORE},
+    {"multithreadcore", HELICS_FLAG_MULTI_THREAD_CORE},
+    {"multiThreadcore", HELICS_FLAG_MULTI_THREAD_CORE},
```

Comment:
```suggestion
    {"single_thread_core", HELICS_FLAG_SINGLE_THREAD_CORE},
    {"singlethreadcore", HELICS_FLAG_SINGLE_THREAD_CORE},
    {"singleThreadCore", HELICS_FLAG_SINGLE_THREAD_CORE},
    {"multi_thread_core", HELICS_FLAG_MULTI_THREAD_CORE},
    {"multithreadcore", HELICS_FLAG_MULTI_THREAD_CORE},
    {"multiThreadCore", HELICS_FLAG_MULTI_THREAD_CORE},
```

I think the camelCase version should match the snake_case variant for splitting up words.

Reply:
I agree good catch!